docs/src/blockchain-development/access_control.md

```diff
@@ -15,3 +15,17 @@ The `msg_sender` function works as follows:
 - If the caller is a contract, then `Result::Ok(Sender)` is returned with the `ContractId` sender variant.
 - If the caller is external (i.e. from a script), then all coin input owners in the transaction are checked. If all owners are the same, then `Result::Ok(Sender)` is returned with the `Address` sender variant.
 - If the caller is external and coin input owners are different, then the caller cannot be determined and a `Result::Err(AuthError)` is returned.
+
+## Using `BASE_ASSET_ID` as deafult/null address
+
+The `BASE_ASSET_ID` constant is **not** a reserved address and should not be treated as such.
+
+`BASE_ASSET_ID` is defined in the Standard Library. It is a `b256` type and is eqivalent to the 0 value.
```

Comment:
1. `BASE_ASSET_ID` will be a `ContractId` soon: #2020. Might as well do the doc update now, but you could defer it to a future PR if you want.
2. The value of `BASE_ASSET_ID` should never actually be needed by users. It could be `0xddec0e7e6a9a4a4e3e57d08d080d71a299c628a46bc609aab4627695679421ca` for all they know.
```suggestion
`BASE_ASSET_ID` is defined in the standard library as a `ContractId` type whose value should not be depended on.
```